nupic/research/frameworks/pytorch/imagenet/network_utils.py

```diff
@@ -60,7 +57,7 @@ def init_resnet50_batch_norm(model):
             m.weight.data.normal_(0, 0.01)
 
 
-def get_compatible_state_dict(model, state_dict):
+def get_compatible_state_dict(state_dict, model):
```

Comment:
Why do you need to reorder the args of this function?

Reply:
This was to make them consistent with the `state_dict_transform` passed into the checkpoint loaded. Such transforms, I believe, are primarily operating over the state_dict, and using the model to supplement the logic.

I also have used `state_dict_transform` in more than two places other than `get_compatible_state_dict`, so it was much simpler to refactor this particular function.

Reply:
What about `resize_model_buffers`? it is using the reversed order, I think this could cause some confusion. See code below:
https://github.com/numenta/nupic.research/pull/329/files/ba12ca407c2e353c6198f79da6acffd101b91e7e#diff-71a69a5efd3d0a6395682e342c951ccfR114-R121
```
  # Resize the buffers of the model to match those in the state_dict.
    if resize_buffers:
        resize_model_buffers(model, state_dict)  # done in place

    # Apply any custom transformation.
    if state_dict_transform:
        state_dict = state_dict_transform(state_dict, model)


```

Reply:
Fair, but it's not transforming the state_dict, it's acting on the model. The model's buffers are in fact being changed.